tsconfig.app.json

```diff
@@ -3,7 +3,7 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "outDir": "./out-tsc/app",
-    "types": []
+    "types": ["node"]
```

Comment:
Why is this needed?

Reply:
@olivierobert To be able to use `process.env` on environment files because `process` is from node